src/openzaak/config/forms.py

```diff
@@ -75,5 +75,4 @@ class Meta:
             "user_representation",
             "header_key",
             "header_value",
-            "oas",
```

Comment:
There is a react form (`external-formset.js`) for this that still has the OAS url as a field, could you remove that as well?

<img width="1420" height="450" alt="image" src="https://github.com/user-attachments/assets/fd301716-cec8-4b0f-8b04-45dc92113f87" />